Sources/allonet2/PlaceServer.swift

```diff
@@ -161,15 +163,21 @@ public class PlaceServer : AlloSessionDelegate
     
     nonisolated public func session(didDisconnect sess: AlloSession)
     {
-        let cid = sess.clientId!
+        guard let cid = sess.clientId else {
```

Comment:
You asked about code style: all my code uses Allman braces (so that {  and } always go on a new line). I'm not super strict about this (e g I might make an exception for an `else`), but FYI that's the general style in this repo.